index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+   <header>
+      <div id="nav_header">
+        <div class="time"><p>9:41</p></div>
+        <div class="space"></div>
+        <div class="cellular_connection">
+        <img src="images/header/tower.png" alt="cellular_image">
+        </div>
+        <div class="wifi">
+        <img src="images/header/Wifi.png" alt="wifi_image">
+        </div>
+        <div class="battery">
+        <img src="images/header/Battery.png" alt="battery">
+        </div>
+      </div>
+      <nav>
+        <div class="nav_logo">
+            <h1>My logo</h1>
+        </div>
+        <div class="navMenu">
+            <img src="images/header/Union.png" alt="menu">
+        </div>
+      </nav>
+   </header>
+
+   <main>
+    <div class="background">
+        <img src="images/header/header.png" alt="background">
+    </div>
+    <section id="content">
+        <div class="heading">
+            <h2>
+                I'm Praveen 
+                Glad to see 
+                you! 
+            </h2>
```

Comment:
- Kindly add break lines to separate the lines. It is for the best practices of HTML & CSS. 👍🏽 We could check this [information](https://www.w3schools.com/Tags/tag_br.asp).